statuspage/src/components/CanaryModal.vue

```diff
@@ -0,0 +1,60 @@
+<!-- This component encapsulates a modal that is displayed -->
+<!-- when clicking on canary                               -->
+<template>
+  <b-modal :id="'modal-canary'+name+namespace" ok-only>
+    <template #modal-title>
+      <img :src="'images/' + checkType + '.svg'" height="30px"> {{name}}
+    </template>
+    <template v-slot:default>
```

Comment:
I don't believe we need this `<template>` node.

Reply:
removed